soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -142,6 +142,30 @@ config SOC_ENABLE_LFXO
 	  to use the LFXO. Otherwise, XL1 and XL2 pins will behave as regular
 	  GPIOs.
 
+choice SOC_ENABLE_LFXO_CAP
```

Comment:
Maybe this can be done in the `soc` node of the device tree? Clock node is probably appropriate:

```
clock: clock@xxxx {
   lfco-cap: "external"; // or "6pf", "7pf", "9pf"
}
```

Reply:
That may require a different node because it is not common between net/app cores.

Reply:
I suggest to not change the device tree at this point - It will drag in more changes.
For instance, we should move more parts of the configuration into the device tree:
- LFCLK source
- PPM
- Capacitors

in addition we will need to split the device tree for nRF52 and nRF53, as the options are different for the MCU cores and families.